man1/aur-rfilter.1

```diff
@@ -9,7 +9,7 @@ aur-rfilter \- filter packages in the Arch Linux repositories
 .YS
 
 .SH DESCRIPTION
-\fBaur-rfilter\fR checks if \fIpkgname\fR (provided on stdin) is in the
```

Comment:
Bold is correct here (other man pages should use it as well). cf. `man grep`

Reply:
Should references to other man pages also be bold? I've seen both forms. e.g. $man man; has a couple of them.